agent/functional_tests/util/utils_unix.go

```diff
@@ -238,3 +243,13 @@ func (agent *TestAgent) getBindMounts() []string {
 func (agent *TestAgent) Cleanup() {
 	agent.platformIndependentCleanup()
 }
+
+// AttributesToMap transforms a list of key, value attributes to return a map
+func AttributesToMap(attributes []*ecs.Attribute) map[string]string {
```

Comment:
This is a general function not specific to linux platform, you should put in `utils.go`.